prqlc/prqlc-parser/src/error/mod.rs

```diff
@@ -227,121 +229,3 @@ impl<T, E: WithErrorInfo> WithErrorInfo for Result<T, E> {
         self.map_err(|e| e.with_source(source))
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
@m-span did we lose this? A quick ctrl-f doesn't get any hits, am I missing something?

Reply:
i did some other minor refactors in parser while restructuring. Large file with tests -> tests in separate file

Reply:
Sorry, GitHub wasn't showing the diff, found it when searching locally